openedx/features/course_experience/templates/course_experience/course-updates-fragment-proversity.html

```diff
@@ -0,0 +1,55 @@
+## mako
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+from openedx.features.course_experience import course_home_page_title
+%>
+
+<%block name="content">
+<div class="course-updates container" id="course-container">
+    <header class="page-header has-secondary">
+        ## Breadcrumb navigation
+        <div class="page-header-main">
+            <nav aria-label="${_('Course Updates')}" class="sr-is-focusable" tabindex="-1">
+                <div class="has-breadcrumbs">
+                    <div class="breadcrumbs">
+                        <span class="nav-item">
+                            <a href="${course_url}">${course_home_page_title(course)}</a>
+                        </span>
+                        <span class="icon fa fa-angle-right" aria-hidden="true"></span>
+                        <span class="nav-item">${_('Course Updates')}</span>
+                    </div>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <div class="page-content">
+		% if plain_html_updates:
```

Comment:
indent  to the same else level

Reply:
It's already indented. i think it's a text processor problem.
![image](https://user-images.githubusercontent.com/17520199/62157558-1748c780-b2d3-11e9-8433-28c1a057fe2b.png)